core/src/main/scala/caliban/GraphQL.scala

```diff
@@ -120,16 +109,17 @@ trait GraphQL[-R] { self =>
                   validate                              = (doc: Document) =>
                                                             for {
                                                               config       <- Configurator.configuration
+                                                              reqMethod    <- HttpRequestMethod.get
```

Comment:
This call can be moved inside of `prepare`, no?

What are the breaking changes?

Reply:
Yes - I initially had it in there but somehow something feels weird for the behaviour of the `.prepare` method to be dependent on an argument that is provided internally via a `FiberRef`.

Only other breaking change is for `Caliban.validation.Context`, but I guess that's a tolerable one because it's very unlikely that users will be using it

Reply:
There's also the downside that the `HttpRequestMethod` needs to be made public because `Context` is public (unless we use a boolean flag `isGetRequest` instead).

Reply:
Okay let's keep the other PR then